src/lcd/sainsmartks.hpp

```diff
@@ -32,7 +32,7 @@ namespace upm
   /**
    * @library i2clcd
    * @sensor sainsmartks
-   * @comname SainSmart LCD Keypad Shield
+   * @comname SainSmart LCD keypad shield
```

Comment:
Remove SaintSmart, this one might have to be renamed entirely as it works with other analog keypad LCD shields (e.g. DFRobot)